duckduckgo/Android
# ViewBinding migrations (BrowserTabFragment, Cta, ...)

Task/Issue URL: https://app.asana.com/0/488551667048375/1201693370277849/f

### Description
ViewBinding migrations.

This PR includes:
- BrowserTabFragment
- Cta
- TrackerNetworkLeaderboardPillView
- TrackingProtectionAppsAdapter
- ReportBreakageAppListAdapter
- PastWeekTrackerActivityContentView

### Steps to test this PR

_smoke test_
- open the browser
- test the onboarding
- visit any site and add a few favorites
- ensure widget cta works as expected
- ensure favorites shortcuts work as expected
- enable AppTP and validate app trackers history and list of apps inside their settings works
- visit privacy dashboard
- use the app as usual, just to see if we spot any bug coming from this migration
